src/client.js

```diff
@@ -707,6 +711,9 @@ class Client {
                 if (that._traceAPICalls)
                     console.log(`SOAP//failure ${ex.toString()}`);
                 that._notifyObservers((observer) => observer.onSOAPCallFailure && observer.onSOAPCallFailure(soapCall, ex) );
+                // Call session expiration callback in case of XSV-350008 Campaign error (Session expiration)
+                if (ex.errorCode == 'XSV-350008')
```

Comment:
You can use "==="

Reply:
I did a change to use the statusCode instead of the error code